core/src/main/java/nl/nn/adapterframework/webcontrol/api/BrowseQueue.java

```diff
@@ -55,7 +55,7 @@
 	public Response getBrowseQueue() throws ApiException {
 		Map<String, Object> returnMap = new HashMap<String, Object>();
 
-		List<String> jmsRealms=JmsRealmFactory.getInstance().getRegisteredRealmNamesAsList();
+		List<String> jmsRealms=JmsRealmFactory.getInstance().getConnectionFactoryNames();
```

Comment:
You cannot just use a connectionFactoryName as a realm name.
Preferably you update the GUI, so that it is clear that you select a connectionFactory,
and also update the code that uses it

Reply:
Of course :)